service/src/test/java/uk/gov/hmcts/reform/fpl/service/CaseDataExtractionServiceTest.java

```diff
@@ -113,11 +109,8 @@ void shouldMapCaseDataWhenEmptyListValues() throws IOException {
         Map<String, Object> templateData = caseDataExtractionService
             .getStandardOrderDirectionData(caseData);
 
-        assertThat(templateData.get("judgeAndLegalAdvisor")).isEqualTo(ImmutableMap.of(
```

Comment:
(0) I know it's not your code, but not sure why this test scenario is different than above - what benefit it gives us to test half of the fields empty? And repeat the test for all of them empty.

Reply:
agree that it doesn't really have much benefit but we will leave it in as don't want to touch this code in this PR